### PR TITLE
Update getting-started-with-swashbuckle.md

### LINKED ISSUE #31720: Build-time generation not supported for .NET 8, but references to a page without details
### Description

The documentation page says that build-time generation of swagger files is not supported using swashbuckle in .NET 8.
It references the main page for more explanation, but the main page does not detail anything about build-time generation.

We're stuck at the moment how to generate swagger files at build time, as the dotnet tool swagger expects all configuration to be provided, which is not working ofcourse

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-8.0&tabs=visual-studio

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/getting-started-with-swashbuckle.md

### Document ID

59222460-a971-e76e-4fe9-23d4d7d768ee

### Article author

@zuckerthoben

---
[Associated WorkItem - 209083](https://dev.azure.com/msft-skilling/Content/_workitems/edit/209083)

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #31720

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/web-api-help-pages-using-swagger.md](https://github.com/dotnet/AspNetCore.Docs/blob/a61882c144dae1d473d92694efd5e381cb39b0e8/aspnetcore/tutorials/web-api-help-pages-using-swagger.md) | [ASP.NET Core web API documentation with Swagger / OpenAPI](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/web-api-help-pages-using-swagger?branch=pr-en-us-31731) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@dasMulli please review.

**Review by tdykstra — APPROVED** [review]
I don't see how putting that PR in a comment is helpful here.

File: aspnetcore/tutorials/getting-started-with-swashbuckle.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ uid: tutorials/get-started-with-swashbuckle
 > [!NOTE]
 > Build-time OpenAPI document generation with Swashbuckle isn't supported in .NET 8 and later. For a supported build-time alternative, see <xref:tutorials/web-api-help-pages-using-swagger>. Run-time document generation is still supported in .NET 8.
 
+<!-- see https://github.com/dotnet/AspNetCore.Docs/pull/31602 -->
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/getting-started-with-swashbuckle.md:18]:
```suggestion
```

File: aspnetcore/tutorials/web-api-help-pages-using-swagger.md [hunk 2]
```diff
@@ -153,6 +153,10 @@ curl -i -H "Authorization: Bearer {token}" https://localhost:{port}/swagger/v1/s
 
 For more information on testing with JWT tokens, see <xref:security/authentication/jwt>.
 
+## Generate an XML documentation file at compile time.
+
+See [GenerateDocumentationFile](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#generatedocumentationfile) for more information.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/web-api-help-pages-using-swagger.md:158]:
```suggestion
See [GenerateDocumentationFile](/dotnet/core/project-sdk/msbuild-props#generatedocumentationfile) for more information.
```

**Comment by Rick-Anderson** [comment]
> I don't see how putting that PR in a comment is helpful here.

deleted

**Review by dasMulli — APPROVED** [review]

**Pull request merged**